plugins/lang-groovy/src/test/java/org/elasticsearch/messy/tests/TDigestPercentilesTests.java

```diff
@@ -383,22 +364,6 @@ public void testScriptMultiValued() throws Exception {
     }
 
     @Override
-    public void testScriptExplicitMultiValued() throws Exception {
```

Comment:
I removed them because as far as I looked, the impl of testScriptExplicitMultiValued was always the same as testScriptMultiValued. I could add it back, but the removal looked like an easy win to me?